app/models/host/discovered.rb

```diff
@@ -25,6 +26,10 @@ class Host::Discovered < ::Host::Base
   scoped_search :in => :discovery_attribute_set, :on => :disk_count, :rename => :disk_count, :complete_value => true, :only_explicit => true
   scoped_search :in => :discovery_attribute_set, :on => :disks_size, :rename => :disks_size, :complete_value => true, :only_explicit => true
 
+  default_scope lambda {
+    where(taxonomy_conditions).order("hosts.created_at DESC")
+  }
```

Comment:
I understand, this is from Host::Base. I wonder if there is a way to chain default scopes... I guess its not possible.